src/components/Header/Header.scss

```diff
@@ -1,13 +1,8 @@
 @import '~style/variables';
 
-header {
+.header {
```

Comment:
Some erroneous styles directly on tags broke when I added more headers to the site.